rfd/0074-sftp-support.md

```diff
@@ -0,0 +1,83 @@
+---
+authors: Andrew LeFevre (andrew.lefevre@goteleport.com)
+state: draft
+---
+
+# RFD 74 - SFTP Support
+
```

Comment:
Can you add required approvers section? Use `@klizhentas || @xinding33` from product and `@r0mant` from engineering.